gendiff/formats/plain.py

```diff
@@ -10,6 +10,8 @@
 
 
 def convert_value_plain_format(val: Any) -> Any:
+    if type(val) is int and val == 0:
```

Comment:
`if val == 0` will do the same

Reply:
you can also add it into `exceptions_to_convert`

Reply:
sure I tried all this prioritly and  in different ways, but resumed it's absolutely nessecary to include val type (int) explicitly as condition.
Otherwise  vals with initial values False, "", None are converted to 0 
or vals with initial values 0 conver are converted to false.